### ISSUE TITLE
Any plan to support sqlite backend?

### ISSUE THREAD

**Issue by qbx2** [lanpa/tensorboardX#398]
Latest tensorboard release includes sqlite backend support(experimental) with --db flag. Is tensorboardX going to support it when the feature be stable or whenever?

**Comment by lanpa** [comment]
Yes, tensorboardX is under heavy reconstruction in hope that it supports different backends in the future.
